benchmarks/functional_autograd_benchmark/functional_autograd_benchmark.py

```diff
@@ -99,15 +156,32 @@ def get_v_for(model: Callable, inp: InputsType, task: str) -> VType:
 
     return v
 
-def run_once(model: Callable, inp: InputsType, task: str, v: VType) -> None:
+def run_once(model: Callable, inp: InputsType, task: str, v: VType, **kwargs) -> None:
     func = get_task_func(task)
 
     if v is not None:
         res = func(model, inp, v=v, strict=True)
     else:
         res = func(model, inp, strict=True)
 
-def run_model(model_getter: GetterType, args: Any, task: str) -> List[float]:
+def run_once_functorch(model: Callable, inp: InputsType, task: str, v: VType, check_consistency=False) -> None:
+    func = get_task_functorch(task)
+
+    if v is not None:
+        res = func(model, inp, v=v, strict=True)
+    else:
+        res = func(model, inp, strict=True)
+
+    if check_consistency:
```

Comment:
@zou3519 I added a consistency checking here which is done during warm-up

Reply:
Do we need to set the random seed?

Reply:
Actually, no as `model` and `inp` are set up once. But I had to disable dropout using `model.eval()` internally.

Reply:
Hmm, we might need it in the future if someone adds new models that do randomness inside the model, but this sounds good for now